<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualOutputSettingWidget</class>
 <widget class="QWidget" name="VirtualOutputSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Virtual Output Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxEnable">
     <property name="title">
      <string>Enable Virtual Output</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutEnable">
      <item>
       <widget class="QCheckBox" name="checkBoxEnable">
        <property name="toolTip">
         <string>Enable the virtual output window for OBS capture</string>
        </property>
        <property name="text">
         <string>Enable virtual output for OBS capture</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxResolution">
     <property name="title">
      <string>Resolution</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutResolution">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutResolution">
        <item>
         <widget class="QLabel" name="labelResolution">
          <property name="text">
           <string>Resolution Preset:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxResolution">
          <property name="toolTip">
           <string>Select output resolution preset</string>
          </property>
          <item>
           <property name="text">
            <string>720p (1280x720)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1080p (1920x1080)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1440p (2560x1440)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Custom</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutCustomResolution">
        <item>
         <widget class="QLabel" name="labelWidth">
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxWidth">
          <property name="toolTip">
           <string>Custom width in pixels</string>
          </property>
          <property name="minimum">
           <number>320</number>
          </property>
          <property name="maximum">
           <number>7680</number>
          </property>
          <property name="value">
           <number>1920</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelHeight">
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxHeight">
          <property name="toolTip">
           <string>Custom height in pixels</string>
          </property>
          <property name="minimum">
           <number>240</number>
          </property>
          <property name="maximum">
           <number>4320</number>
          </property>
          <property name="value">
           <number>1080</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerResolution">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutTheme">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutThemeSource">
        <item>
         <widget class="QRadioButton" name="radioButtonMirrorDisplay1">
          <property name="toolTip">
           <string>Mirror the theme settings from Display 1</string>
          </property>
          <property name="text">
           <string>Mirror Display 1</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonCustomTheme">
          <property name="toolTip">
           <string>Use a custom theme for virtual output</string>
          </property>
          <property name="text">
           <string>Custom Theme:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxTheme">
          <property name="toolTip">
           <string>Select a custom theme for virtual output</string>
          </property>
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOverlay">
     <property name="title">
      <string>Overlay Image (Logo)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutOverlay">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutOverlay">
        <item>
         <widget class="QLineEdit" name="lineEditOverlayPath">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Path to overlay image file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonBrowseOverlay">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonClearOverlay">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelOverlayPreview">
        <property name="text">
         <string>Preview:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelOverlayImage">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #ccc; background-color: #f0f0f0;</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxLowerThird">
     <property name="title">
      <string>Lower Third</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutLowerThird">
      <item>
       <widget class="QCheckBox" name="checkBoxShowLowerThird">
        <property name="toolTip">
         <string>Show lower third overlay on virtual output</string>
        </property>
        <property name="text">
         <string>Show lower third</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelLowerThirdFont">
        <property name="text">
         <string>Font:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutLowerThirdFont">
        <item>
         <widget class="QFontComboBox" name="fontComboBoxLowerThird">
          <property name="toolTip">
           <string>Select font for lower third text</string>
          </property>
          <property name="writingSystem">
           <enum>QFontDatabase::Any</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonLowerThirdFont">
          <property name="text">
           <string>More...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLowerThirdFontPreview">
          <property name="text">
           <string>Aa Bb Cc</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutLowerThirdColors">
        <item>
         <widget class="QLabel" name="labelLowerThirdBgColor">
          <property name="text">
           <string>Background:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonLowerThirdBgColor">
          <property name="text">
           <string>Color...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsViewLowerThirdBgColor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLowerThirdTextColor">
          <property name="text">
           <string>Text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonLowerThirdTextColor">
          <property name="text">
           <string>Color...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsViewLowerThirdTextColor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerColors">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutLowerThirdPosition">
        <item>
         <widget class="QLabel" name="labelLowerThirdPosition">
          <property name="text">
           <string>Position:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxLowerThirdPosition">
          <property name="toolTip">
           <string>Position of lower third on screen</string>
          </property>
          <item>
           <property name="text">
            <string>Bottom Left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bottom Center</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bottom Right</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerPosition">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxWindow">
     <property name="title">
      <string>Window Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutWindow">
      <item>
       <widget class="QCheckBox" name="checkBoxAlwaysOnTop">
        <property name="toolTip">
         <string>Keep virtual output window always on top</string>
        </property>
        <property name="text">
         <string>Always on top</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutPreview">
        <item>
         <widget class="QPushButton" name="pushButtonPreview">
          <property name="toolTip">
           <string>Preview the virtual output window</string>
          </property>
          <property name="text">
           <string>Preview Window</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerPreview">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
